lib/inferno/dsl/fhir_evaluator/evaluation_result.rb

```diff
@@ -0,0 +1,58 @@
+module FhirEvaluator
+  # The result of a Rule evaluating a data set.
+  class EvaluationResult
+    attr_accessor :message,
+                  :severity,   # fatal | error | warning | information | success
+                  :issue_type, # https://www.hl7.org/fhir/valueset-issue-type.html
+                  :threshold,  # quantitative value that a rule checks for
+                  :value,      # actual observed value
+                  :rule        # Rule that produced this result
+
+    def initialize(message, severity: 'warning', issue_type: 'business-rule', threshold: nil, value: nil, rule: nil)
```

Comment:
Is there a reason for these severity and issue_type defaults?

Reply:
Just convenience really. We can certainly remove or reconsider these, but I expected these would be the values for nearly every "issue" the evaluator finds